cosmos-admin/README.md

```diff
@@ -35,15 +46,168 @@ If for any reason the data copying is interrupted (for instance, the communicati
 
 [Top](#top)
 
+###<a name="getuserstats"></a>`get_user_stats.sh`
+
+This script has been designed to get certain user statistics, these ones:
+
+* Last access time (timestamp format).
+* HDFS usage (number of bytes).
+* Local file system usage (number of bytes).
+* Number of successful ssh connections.
+* Number of failed ssh connections.
+
+The above values are stored within a database usally named `cosmos`,  in a table usually named `cosmos_user`. Follow [this link](../cosmos-gui/README.md#database) for more details about the database.
+
+`get_user_stats.sh` is parameterized by:
+
+* Host running the MySQL server
+* Port where the MySQL server listens for requests. Typically, TCP/3306 port.
+* Database name. Usually, `cosmos`.
+* MySQL user allowd to insert data within the `cosmos` database, `cosmos_user` table.
+* Pasword for the MySQL user.
+* ssh log file (usually, `/var/log/secure`).
+
+In a shell:
+
+    $ ./get_user_stats.sh <mysql_host> <mysql_port> <db_name> <mysql_user> <mysql_password> <ssh_logs>
+
+Since the above mentioned statistics change along the time, it is useful to schedule this script execution in a periodic fashion, let's say at least once per day. You can use the `crontab` for achieving this.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/5c07eed2af6cdb9c7cb9b61aaf34991335670db2